### PR TITLE
Adding ability to specify a classic build

### PR DESCRIPTION
**Pull request by blckmn**
This means a user rather than using cloud options, can simply get all options (same as previous unified nightlies).

### CODE REVIEW

File: src/js/tabs/firmware_flasher.js [hunk 1]
```diff
@@ -647,6 +647,16 @@ firmware_flasher.initialize = function (callback) {
             $('input.flash_manual_baud').prop('checked', false);
         }
 
+        $('input.classicbuild_mode').change(function () {
+            const status = $(this).is(':checked');
+
+            $('select[name="radioProtocols"').enabled = !status;
+            $('select[name="telemetryProtocols"').enabled = !status;
+            $('select[name="motorProtocols"').enabled = !status;
+            $('select[name="options"').enabled = !status;
```
Comment (haslinghuis) [inline src/js/tabs/firmware_flasher.js:656]:
```suggestion
            $('select[name="radioProtocols"').prop("disabled", status);
            $('select[name="telemetryProtocols"').prop("disabled", status);
            $('select[name="motorProtocols"').prop("disabled", status);
            $('select[name="options"').prop("disabled", status);
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: src/js/tabs/firmware_flasher.js (hunk 1 above)
Comment (blckmn) [reply src/js/tabs/firmware_flasher.js]: haha - just noticed that and corrected it :)

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Comment by HThuren** [comment]
Great idea to have this classic switch, but maybe the options choosen at classic build better was same as the one in ex. 4.3.1 where you found baro and mag included.

**Comment by haslinghuis** [comment]
@HThuren In classic build the build options are disabled and would contain the whole firmware set as before.

**Pull request merged**